src/Sylius/Component/Core/Provider/ProductVariantsPricesProvider.php

```diff
@@ -23,8 +23,9 @@
 
 final class ProductVariantsPricesProvider implements ProductVariantsPricesProviderInterface
 {
-    public function __construct(private ProductVariantPriceCalculatorInterface $productVariantPriceCalculator)
-    {
+    public function __construct(
+        private ProductVariantDataMapProviderInterface $productVariantDataMapProvider,
```

Comment:
The previous argument should be deprecated and the change mentioned in the upgrade file.

Reply:
Went a different route and deprecated the whole old flow.